### DISCUSSION TITLE
Changing windows size

### DISCUSSION THREAD

**Discussion by mhambuch** [mworion/MountWizzard4#288] [Q&A] [answered]
Hi Michel, 

Not an Issue but i did not  knew where  to  post 

I Just reinstalled mw4 from scratch  using the 3.x install  procedure and calling  startup.pyz 
Installation went  well,  i start  mw4  calling startup.pyz again. When opening  windows  within  the app,  (eg messages, horizon) i cannot resize those nor change fonts. In the past  there were scripts to start and update mw4 in which font size could be indicated.

Many thanks.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by mworion** [comment, marked as answer]
Hi,
moved it to discussions.
There were still the options as parameters when calling startup.pyz.
In this case you should make your batch file with:

```
python startup.pyz --dpi x --scale y
```
where you put for x and y the parameters you have used in your v2.0 script

```
python startup.pyz --help 
```

gives you all options.
Hope this helps,
Michel

**Comment by mhambuch** [comment]
Many Thanks - I created a bat file to start mw4 scale works fine but dpi is not (same font size whatever value i tried from 1 to 400)

![image](https://user-images.githubusercontent.com/63575440/210131405-22015df7-f382-4257-9ee1-6ddfed93afa1.png)

**Comment by mworion** [comment]
Still no clue why this happens. As intermediate, you could also start v3 with your v2 start scripts. There, the option seem to work.
Michel

**Comment by mworion** [comment]
Ok I think I found it. Please download the new startup script (should be v3.0beta7) and use the parameters as intended. For some reasons QT does only like integer numbers for DPI, no float even when float is '48.0'. 
Please report back if that helps
Michel

**Comment by mhambuch** [comment]
Hi Michel,

Works with the V30beta7, many thanks!

![image](https://user-images.githubusercontent.com/63575440/210213926-aefae592-d13f-4ccd-9b0d-304062947414.png)

**Comment by mworion** [comment]
The positioning of the windows is not impacted by the starting scripts. If you shift the windows to far away out of sight, I get them back. But this was introduced during v2 of MW4. Could you explain a little more, what you want to experience ?

**Reply by mhambuch** [reply]
Hi Michel, everything is working fine now,  i first thought that mw4 was not remembering where i placed the windows on each previous session when starting mw4 again,  but it  does, all good. Many thanks for your always Quick answers, and all the Best for 2023.

**Reply by mworion** [reply]
All the best for you as well !
